.github/CODEOWNERS

```diff
@@ -4,9 +4,9 @@
 .circleci/                    @napetrov @Alexsandruss @homksei @ahuber21 @ethanglaser
 
 # Docs
-*.md                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust
-doc/                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust
-requirements-doc.txt          @emmwalsh @Alexsandruss
+*.md                          @maria-Petrova @napetrov @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
+doc/                          @maria-Petrova @napetrov @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
+requirements-doc.txt          @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
```

Comment:
Minor comment but Samir and Alexander would already be covering requirements-doc.txt from line 20

Reply:
The last matching pattern takes the most precedence. From my understanding, when someone opens a pull request that only modifies `requirements-doc.txt` file, only the owners from line 9 and not the global owner(s) for the `requirements*` from L20 will be requested for a review.